packages/embeds/embed-core/src/embed.ts

```diff
@@ -116,6 +116,31 @@ function validate(data: Record<string, unknown>, schema: ValidationSchema) {
   }
 }
 
+function getColorScheme(el: HTMLElement) {
+  const pageColorScheme = getComputedStyle(el).colorScheme;
```

Comment:
Allows us to get the color scheme  from any element(even when it is set on root element). It allows us to handle all scenarios of setting color-scheme even down the DOM tree.